tests/pytorch/nightly/mnist-3-7.libsonnet

```diff
@@ -1,66 +0,0 @@
-// Copyright 2020 Google LLC
```

Comment:
why is this config deleted?

Reply:
Previously, all tests except for this one (and some of the CUDA tests) were running on the pytorch XLA `nightly` image with Python 3.6. This PR changes the image for all of the tests to the`nightly_3.7` Docker image that `mnist-3-7` was already using, making is obsolete.